src/main/webapp/app/app.js

```diff
@@ -21,7 +21,8 @@ angular.module('trialdirect', [
 
                 $rootScope.$on("$stateChangeError", console.log.bind(console));
 
-            }
+                // The line above mimics an inadvertant deletion to highlight how removed lines are illustrated in Github
+                console.log('This console log mimics a debug log statement left over after implementation');
```

Comment:
You should remove the console.log statement left over as part of your debugging process.